x-pack/plugins/data_enhanced/config.ts

```diff
@@ -12,7 +12,8 @@ export const configSchema = schema.object({
       enabled: schema.boolean({ defaultValue: false }),
       pageSize: schema.number({ defaultValue: 10000 }),
       trackingInterval: schema.duration({ defaultValue: '10s' }),
-      inMemTimeout: schema.duration({ defaultValue: '1m' }),
+      onScreenTimeout: schema.duration({ defaultValue: '5m' }),
```

Comment:
Yes, because otherwise, any completed sessions, will still stay in memory of the `.async_search` index.
This index is heavy and users won't be happy about it.